website/docs/docs/dbt-versions/release-notes/03-Oct-2023/api-v2v3-limit.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "API results limited to `100`"
+id: apiv3-limit"
+description: "Oct 2023: In order to enhance the efficiency and stability of our services, we will limit all API results to `100` records. This limit is applicable to multi-tenant instances only."
+sidebar_label: "Update: API results limited to `100`"
+sidebar_position: 04
+tags: [Oct-2023, API]
+---
+
+
+Beginning December 1, 2023, the [Administrative API](/docs/dbt-cloud-apis/admin-cloud-api) v2 and v3 will limit all "list" or `GET` API methods to 100 results per API request. This limit enhances the efficiency and stability of our services. You can handle more than 100 results by using the `limit` and `offset` query paramters to paginate those results. Otherwise, exceeding the limit of `100` will cause an error. 
```

Comment:
```suggestion
Beginning December 1, 2023, the [Administrative API](/docs/dbt-cloud-apis/admin-cloud-api) v2 and v3 will limit all "list" or `GET` API methods to 100 results per API request. This limit enhances the efficiency and stability of our services. You need to handle more than 100 results by using the `limit` and `offset` query parameters to paginate those results; otherwise, you will receive an error. 
```